libres/lib/analysis/update.cpp

```diff
@@ -56,21 +56,21 @@ void ensure_node_loaded(const enkf_config_node_type *config_node,
 
 void serialize_node(enkf_fs_type *fs, const enkf_config_node_type *config_node,
                     int iens, int row_offset, int column,
-                    const ActiveList *active_list, matrix_type *A) {
+                    const ActiveList *active_list, Eigen::MatrixXd &A) {
 
     enkf_node_type *node = enkf_node_alloc(config_node);
     node_id_type node_id = {.report_step = 0, .iens = iens};
-    enkf_node_serialize(node, fs, node_id, active_list, A, row_offset, column);
+    enkf_node_serialize(node, fs, node_id, active_list, &A, row_offset, column);
```

Comment:
Is the "&" in &A needed here?

Reply:
yes since, enkf_node_serialize does expect a matrix pointer